### PR TITLE
Update conn.go

### PR DESCRIPTION
**Pull request by johnnadratowski**

### CODE REVIEW

File: conn.go [hunk 1]
```diff
@@ -356,12 +356,6 @@ func (c *boltConn) Close() error {
 		return nil
 	}
 
-	if c.transaction != nil {
```
Comment (robbert229) [inline conn.go]: why are we no longer rolling back the transaction?

Comment (johnnadratowski) [reply conn.go]: This was a duplicate condition.

Comment (johnnadratowski) [reply conn.go]: from #29

Comment (robbert229) [reply conn.go]: cool.

**Pull request merged**